ete3/coretype/tree.py

```diff
@@ -1552,7 +1552,7 @@ def sort_descendants(self, attr="name"):
             if not n.is_leaf():
                 n.children.sort(key=lambda x: str(sorted(node2content[x])))
 
-    def get_cached_content(self, store_attr=None, container_type=set, _store=None):
+    def get_cached_content(self, store_attr=None, container_type=set, _store=None, descendant_leaves=True):
```

Comment:
We need a couple of lines in the unittest to cover this functionality. You can add it around https://github.com/etetoolkit/ete/blob/fb5a1edc0dbaa83b87150493f719470a977a08dc/ete3/test/test_tree.py#L655